rb/lib/selenium/webdriver/chrome/options.rb

```diff
@@ -155,30 +181,50 @@ def headless!
         # @param [String] user_agent Full user agent
```

Comment:
Do we need to change YARD now that you use double-splat parameter?

Reply:
actually, I don't think so? We're still spelling these things out in #initialize and I think it's ok to specify in the comments exactly which opts are supported

Reply:
Right, but should this now be changed to:

```ruby
# @param [Hash] opts
# @option opts [String] :user_agent Full user agent
# ...
```

Reply:
It actually complains on these docs:

```
$ ./go //rb:docs
...
[warn]: @param tag has unknown parameter name: device_name
    in file `build/rb/lib/selenium/webdriver/chrome/options.rb' near line 165
[warn]: @param tag has unknown parameter name: device_metrics
    in file `build/rb/lib/selenium/webdriver/chrome/options.rb' near line 165
[warn]: @param tag has unknown parameter name: user_agent
    in file `build/rb/lib/selenium/webdriver/chrome/options.rb' near line 165
```